packages/astro/src/content/content-layer.ts

```diff
@@ -138,8 +138,14 @@ export class ContentLayer {
 	async #doSync(options: RefreshContentOptions) {
 		const contentConfig = globalContentConfigObserver.get();
 		const logger = this.#logger.forkIntegrationLogger('content');
+
+		if (contentConfig?.status === 'error') {
+			logger.error(`Error loading content config. Skipping sync.\n${contentConfig.error.message}`);
+			return;
+		}
+
 		if (contentConfig?.status !== 'loaded') {
```

Comment:
Does this check pass when there is no content config? I’d guess a project without a config shouldn’t error/log anything?

Reply:
Good question! Yes, it does pass. It shows as loaded, but with no collections. I'll add a comment.

Reply:
```suggestion
		// It shows as loaded with no collections even if there's no config
		if (contentConfig?.status !== 'loaded') {
```